safe-global/web-core
# fix: omit connection center in mobile devices

## What it solves

Resolves #912

## How this PR fixes it
Renders PairingDetails conditionally on the viewport width.

## How to test it
1. Emulate a mobile viewport in the browser and observe there is no Pairing in the ConnectionCenter nor in the "connect wallet & select network" (step 1) from create Safe flow.
2. In non-mobile viewports the layout should remain the same.

## Analytics changes
N/A

## Screenshots
<img width="884" alt="Screenshot 2022-10-18 at 16 55 10" src="https://user-images.githubusercontent.com/32431609/196466079-14a6619c-84bd-443d-9bcd-1af7e1a1e2f0.png">
<img width="468" alt="Screenshot 2022-10-19 at 17 57 57" src="https://user-images.githubusercontent.com/32431609/196743915-d8fb8abf-41f9-4153-a55b-ff66118002b5.png">
